boards/arduino-leonardo/examples/leonardo-adc.rs

```diff
@@ -10,12 +10,13 @@ fn main() -> ! {
     let dp = arduino_leonardo::Peripherals::take().unwrap();
 
     let mut pins = arduino_leonardo::Pins::new(dp.PORTB, dp.PORTC, dp.PORTD, dp.PORTE, dp.PORTF);
+    let baudrate = 57600.into_baudrate();
 
     let mut serial = arduino_leonardo::Serial::new(
         dp.USART1,
         pins.d0,
         pins.d1.into_output(&mut pins.ddr),
-        57600,
+        baudrate,
     );
```

Comment:
The variable here is superfluous, just inline the expression into the function call:

```suggestion
    let mut serial = arduino_leonardo::Serial::new(
        dp.USART1,
        pins.d0,
        pins.d1.into_output(&mut pins.ddr),
        57600.into_baudrate(),
    );
```

The same goes for all the other examples.